packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -37,20 +37,38 @@ const contextTypes = {
 
 
 class CollapsibleMenuViewItemGroup extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleOnChange = this.handleOnChange.bind(this);
+  }
+
   getChildContext() {
     return { isCollapsibleGroupItem: true };
   }
 
+  handleOnChange(event, selectedIndex) {
+    if (this.props.onChange) {
+      let selectedKey = '';
+      React.Children.forEach(this.props.children, (child, index) => {
+        if (selectedIndex === index) {
+          selectedKey = child.key;
```

Comment:
Perhaps we can use the find method here instead? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
You can't use array functions with children, they don't work when there are less than 2 children.